ckanext/canada/templates/public/package/snippets/schemaorg.html

```diff
@@ -0,0 +1,91 @@
+          {%- else -%}
+            {%- set license_url, license_title = data.license_url, data.license_title -%}
+          {%- endif -%}
+
+       <span property="name" value="{{ license_title }}"></span>
+            <span property="sameAs" value="{{ license_url }}"></span>
+            <span property="text" value="{{ _('Rights under which the catalog can be reused are outlined in the Open Government Licence - Canada') }}"></span>
+            </span>
+        </div>
+
+        <span property="name">{{ h.get_translated(data, 'title') }}</span>
+        <span property="description">{{ h.get_translated(data, 'notes') }}</span>
+        <span property="datePublished">{{ data.date_published[:10] }}</span>
+        <span property="dateModified">{{ data.metadata_modified[:10] }}</span>
+        <span property="publisher" typeof="Organization">
+    <span property="name">{{ cr.split_bilingual_field(data.organization.title, h.lang()) }}</span>
+        <span property="email">{{ data.maintainer_email }}</span>
+        </span>
+        {%- if data.get('time_period_coverage_start') -%}
+          {%- set tempo_coverage= '/'.join([data.get('time_period_coverage_start', '')[:10], data.get('time_period_coverage_end', '')[:10]])  -%}
+          <span property="temporalCoverage" value="{{ tempo_coverage }}"></span>
+        {%- endif -%}
+        {%- set scheming_choices=h.scheming_get_preset('canada_subject').choices -%}
+        {%- for value in data.subject -%}
+          <span property="genre">{{ h.scheming_choices_label(scheming_choices, value) }}</span>
+        {%- endfor -%}
+
+        {%- for keyword in h.get_translated(data.keywords, h.lang()) -%}
+            <span property="keywords">{{ keyword }}</span>
+        {%- endfor -%}
+        <span property="sameAs" value="{{ h.url_for(controller='package', action='read', id=data.id) }}"></span>
+            {%- set schema = h.scheming_get_dataset_schema(data.type) -%}
+            {%- set name_field = h.scheming_field_by_name(schema.resource_fields, 'name_translated') -%}
+            {%- set format_field = h.scheming_field_by_name(schema.resource_fields, 'format') -%}
+            {%- set lang_field = h.scheming_field_by_name(schema.resource_fields, 'language') -%}
+          {%- if data.resources -%}
+            {%- set nonapps = [] -%}
+            {%- for resource in data.resources -%}
+              {%- if not resource.related_type -%}
+                {%- do nonapps.append(resource) -%}
+              {%- endif -%}
+            {%- endfor -%}
+            {%- if nonapps -%}
+              {% for res in nonapps %}
+                <span property="distribution" typeof="DataDownload">
+                    <span property="inLanguage" content="{{ ','.join(res[lang_field.field_name]) }}"> </span>
+                    <span property="name" lang="{{ h.lang() }}">{{ h.scheming_language_text(res[name_field.field_name]) }}</span>
```

Comment:
might as well use `h.get_translated(res, 'name')` instead of scheming_field_by_name and scheming_language text. That's what the core ckan and the other templates do.